dockers/docker-pde/supervisord.conf

```diff
@@ -0,0 +1,12 @@
+[supervisord]
+logfile_maxbytes=1MB
+logfile_backups=2
+nodaemon=true
+
+[program:rsyslogd]
+command=/bin/bash -c "rm -f /var/run/rsyslogd.pid && /usr/sbin/rsyslogd -n"
+priority=1
+autostart=true
+autorestart=false
+stdout_logfile=syslog
+stderr_logfile=syslog
```

Comment:
We recently added a new supervisord-dependent-startup package and also modified supervisor.conf files to start rsyslogd without a PID file. Can you please use PR https://github.com/Azure/sonic-buildimage/pull/4599 as a guide and apply changes to this supervisor.conf file as appropriate?

Reply:
Thanks Joe, updated supervisord.conf.